app/assets/javascripts/libs/error_handling.js

```diff
@@ -30,18 +29,20 @@ class ErrorHandling {
     this.throwAssertions = options.throwAssertions;
     this.sendLocalErrors = options.sendLocalErrors;
 
-    this.commitHash = $("meta[name='commit-hash']").attr("content");
+    this.commitHash = document
+      .querySelectorAll("meta[name='commit-hash']")[0]
```

Comment:
If the meta tag does not exist for some reason (if something's messed up in the build process), this will throw an error and airbrake won't be initialized yet (so we won't notice that). Can you add an extra check to validate that the `[0]' access is safe?

Reply:
I am not sure how to proceed on this one. If the meta data is not present, we cannot initialise airbrake and will not be notified by it. But there is no way around that. Without a valid Airbrake config (including the API key) airbrake can't help us.

Reply:
Isn't this `document.querySelector("meta[name='commit-hash']")`?